co-noir/co-builder/src/flavours/eccvm_flavour.rs

```diff
@@ -0,0 +1,1469 @@
+    const MSM_COLLISION_X2: usize = 34; // column 34
+    const MSM_COLLISION_X3: usize = 35; // column 35
+    const MSM_COLLISION_X4: usize = 36; // column 36
+    const MSM_LAMBDA1: usize = 37; // column 37
+    const MSM_LAMBDA2: usize = 38; // column 38
+    const MSM_LAMBDA3: usize = 39; // column 39
+    const MSM_LAMBDA4: usize = 40; // column 40
+    const MSM_SLICE1: usize = 41; // column 41
+    const MSM_SLICE2: usize = 42; // column 42
+    const MSM_SLICE3: usize = 43; // column 43
+    const MSM_SLICE4: usize = 44; // column 44
+    const TRANSCRIPT_RESET_ACCUMULATOR: usize = 45; // column 45
+    const LOOKUP_READ_COUNTS_0: usize = 46; // column 46
+    const LOOKUP_READ_COUNTS_1: usize = 47; // column 47
+    const TRANSCRIPT_BASE_INFINITY: usize = 48; // column 48
+    const TRANSCRIPT_BASE_X_INVERSE: usize = 49; // column 49
+    const TRANSCRIPT_BASE_Y_INVERSE: usize = 50; // column 50
+    const TRANSCRIPT_ADD_X_EQUAL: usize = 51; // column 51
+    const TRANSCRIPT_ADD_Y_EQUAL: usize = 52; // column 52
+    const TRANSCRIPT_ADD_LAMBDA: usize = 53; // column 53
+    const TRANSCRIPT_MSM_INTERMEDIATE_X: usize = 54; // column 54
+    const TRANSCRIPT_MSM_INTERMEDIATE_Y: usize = 55; // column 55
+    const TRANSCRIPT_MSM_INFINITY: usize = 56; // column 56
+    const TRANSCRIPT_MSM_X_INVERSE: usize = 57; // column 57
+    const TRANSCRIPT_MSM_COUNT_ZERO_AT_TRANSITION: usize = 58; // column 58
+    const TRANSCRIPT_MSM_COUNT_AT_TRANSITION_INVERSE: usize = 59; // column 59
+    // HERE THE SHIFTED START
+    const TRANSCRIPT_MUL: usize = 60; // column 60
+    const TRANSCRIPT_MSM_COUNT: usize = 61; // column 61
+    const PRECOMPUTE_SCALAR_SUM: usize = 62; // column 62
+    const PRECOMPUTE_S1HI: usize = 63; // column 63
+    const PRECOMPUTE_DX: usize = 64; // column 64
+    const PRECOMPUTE_DY: usize = 65; // column 65
+    const PRECOMPUTE_TX: usize = 66; // column 66
+    const PRECOMPUTE_TY: usize = 67; // column 67
+    const MSM_TRANSITION: usize = 68; // column 68
+    const MSM_ADD: usize = 69; // column 69
+    const MSM_DOUBLE: usize = 70; // column 70
+    const MSM_SKEW: usize = 71; // column 71
+    const MSM_ACCUMULATOR_X: usize = 72; // column 72
+    const MSM_ACCUMULATOR_Y: usize = 73; // column 73
+    const MSM_COUNT: usize = 74; // column 74
+    const MSM_ROUND: usize = 75; // column 75
+    const MSM_ADD1: usize = 76; // column 76
+    const MSM_PC: usize = 77; // column 77
+    const PRECOMPUTE_PC: usize = 78; // column 78
+    const TRANSCRIPT_PC: usize = 79; // column 79
+    const PRECOMPUTE_ROUND: usize = 80; // column 80
+    const PRECOMPUTE_SELECT: usize = 81; // column 81
+    const TRANSCRIPT_ACCUMULATOR_EMPTY: usize = 82; // column 82
+    const TRANSCRIPT_ACCUMULATOR_X: usize = 83; // column 83
+    const TRANSCRIPT_ACCUMULATOR_Y: usize = 84; // column 84
+    const Z_PERM: usize = 85; // column 85
+    const LOOKUP_INVERSES: usize = 86; // column 1
```

Comment:
```suggestion
    const LOOKUP_INVERSES: usize = 86; // column 86
```
Or did I miss something?